index.html

```diff
@@ -38,13 +38,13 @@ <h1 class="heading__text">
     </form>
     <section class="drinks">
       <div class="wrapper drinks__wrapper">
-        <!-- HTML Validator recommends adding a heading here. Something to consider. -->
-        <div class="drink__container" data-carousel>
-          <i class="drink__button drink__button--hidden fa-solid fa-caret-left" data-button="prev"></i>
-          <ul class="drink__list" data-slides>
+        <h2 class="drinks__heading"></h2>
```

Comment:
yeah, I used h2 on popup section.
if it's unnecessary to have an empty h2 here, we can just remove it.